be/Model/Entity/User.php

```diff
@@ -48,16 +49,17 @@ class User
      */
     private $deleted;
 
-    public function toArray(): array
-    {
-        return [
-            'userNid' => $this->getUserNid(),
-            'code' => $this->getCode(),
-            'name' => $this->getName(),
-            'rating' => $this->getRating(),
-            'team' => $this->getTeam(),
-        ];
-    }
+    /**
+     * @ORM\OneToMany(targetEntity="Model\Entity\Game", mappedBy="winnerUser")
```

Comment:
Tutaj tak się zastanwiam na ile to jest optymalne. Bo teraz pobierająć userów zawsze są pobierane też ich wszystkie gry (później dopiero w php ucinam do gier z ostatnich 3 dni). Narazie jak nie ma tyle danych to jest spoko, ale jak by było w pyte gier to pewnie by zamulało ? Ciekawe co wtedy powinno się robić? Trzeba wtedy pisać zapytanie sql nie działające na encjach ?

Reply:
Znaczy tak. Moim zdaniem to te gry nie są od razu pobierane przy pobieraniu użytkowników. Żeby je pobrać z bazy, to musisz dopiero skorzystać z gettera tej zmiennej instancyjnej. Wtedy Doctrine je załapuje tak zwanym "lazy loadingiem". Jeśli chcesz tą listę ograniczyć, ażeby nie pobierało z początku istnienia aplikacji, to istnieje coś takiego jak "filter", które jest metodą w klasie `ArrayCollection`. Dla tego typu przypadków tworzy się więc osobny getter.